
[Tue, 09 Jun 2015 18:36:04 -0600] ERROR IN $fillManager->fillReport(). CLAIM ID: 2586

[Tue, 09 Jun 2015 18:36:04 -0600] [[o:JRException]:"net.sf.jasperreports.engine.JRException: Unable to get value for field 'claimant_birth_date' of class 'java.sql.Date'" at:
#-19 net.sf.jasperreports.engine.JRResultSetDataSource.getFieldValue(JRResultSetDataSource.java:319)
#-18 net.sf.jasperreports.engine.fill.JRFillDataset.setOldValues(JRFillDataset.java:1330)
#-17 net.sf.jasperreports.engine.fill.JRFillDataset.next(JRFillDataset.java:1231)
#-16 net.sf.jasperreports.engine.fill.JRFillDataset.next(JRFillDataset.java:1207)
#-15 net.sf.jasperreports.engine.fill.JRBaseFiller.next(JRBaseFiller.java:1549)
#-14 net.sf.jasperreports.engine.fill.JRVerticalFiller.fillReport(JRVerticalFiller.java:149)
#-13 net.sf.jasperreports.engine.fill.JRBaseFiller.fill(JRBaseFiller.java:909)
#-12 net.sf.jasperreports.engine.fill.JRBaseFiller.fill(JRBaseFiller.java:822)
#-11 net.sf.jasperreports.engine.fill.JRFiller.fill(JRFiller.java:61)
#-10 net.sf.jasperreports.engine.JasperFillManager.fill(JasperFillManager.java:446)
#-9 net.sf.jasperreports.engine.JasperFillManager.fillReport(JasperFillManager.java:849)
#-8 sun.reflect.GeneratedMethodAccessor5246.invoke(Unknown Source)
#-7 sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
#-6 java.lang.reflect.Method.invoke(Method.java:597)
#-5 php.java.bridge.JavaBridge.Invoke(JavaBridge.java:1044)
#-4 php.java.bridge.Request.handleRequest(Request.java:417)
#-3 php.java.bridge.Request.handleRequests(Request.java:500)
#-2 php.java.bridge.http.ContextRunner.run(ContextRunner.java:145)
#-1 php.java.bridge.ThreadPool$Delegate.run(ThreadPool.java:60)
#0 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(232): java_ExceptionProxyFactory->getProxy(19, 'net.sf.jasperre...', 'T', true)
#1 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(360): java_Arg->getResult(true)
#2 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(366): java_Client->getWrappedResult(true)
#3 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(560): java_Client->getResult()
#4 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(1752): java_Client->invokeMethod(17, 'getCause', Array)
#5 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(2062): java_JavaProxy->__call('getCause', Array)
#6 /home/claim-sftp/webroot/lib/claim_distribution.php(1980): java_exception->__call('getCause', Array)
#7 /home/claim-sftp/webroot/lib/claim_distribution.php(1980): java_InternalException->getCause()
#8 /home/claim-sftp/webroot/lib/claim_distribution.php(1577): ClaimDistribution->claim2pdf('TX', '2586', '244')
#9 /home/claim-sftp/webroot/cron/distribution_queue.php(166): ClaimDistribution->makeDistribution('TX', '70', '2586', '244', '1', Array, false)
#10 {main}]
[Tue, 09 Jun 2015 18:36:04 -0600] ERROR IN $exportManager->exportReportToPdfFile(). CLAIM ID: 2586
[Tue, 09 Jun 2015 18:36:04 -0600] [[o:NullPointerException]:"java.lang.NullPointerException" at:
#-13 java.io.File.<init>(File.java:222)
#-12 net.sf.jasperreports.engine.util.JRLoader.loadObjectFromFile(JRLoader.java:89)
#-11 net.sf.jasperreports.engine.JasperExportManager.exportToPdfFile(JasperExportManager.java:125)
#-10 net.sf.jasperreports.engine.JasperExportManager.exportReportToPdfFile(JasperExportManager.java:485)
#-9 sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
#-8 sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
#-7 sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
#-6 java.lang.reflect.Method.invoke(Method.java:597)
#-5 php.java.bridge.JavaBridge.Invoke(JavaBridge.java:1044)
#-4 php.java.bridge.Request.handleRequest(Request.java:417)
#-3 php.java.bridge.Request.handleRequests(Request.java:500)
#-2 php.java.bridge.http.ContextRunner.run(ContextRunner.java:145)
#-1 php.java.bridge.ThreadPool$Delegate.run(ThreadPool.java:60)
#0 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(232): java_ExceptionProxyFactory->getProxy(22, 'java.lang.NullP...', 'T', true)
#1 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(360): java_Arg->getResult(true)
#2 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(366): java_Client->getWrappedResult(true)
#3 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(560): java_Client->getResult()
#4 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(1752): java_Client->invokeMethod(21, 'getCause', Array)
#5 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(2062): java_JavaProxy->__call('getCause', Array)
#6 /home/claim-sftp/webroot/lib/claim_distribution.php(2002): java_exception->__call('getCause', Array)
#7 /home/claim-sftp/webroot/lib/claim_distribution.php(2002): java_InternalException->getCause()
#8 /home/claim-sftp/webroot/lib/claim_distribution.php(1577): ClaimDistribution->claim2pdf('TX', '2586', '244')
#9 /home/claim-sftp/webroot/cron/distribution_queue.php(166): ClaimDistribution->makeDistribution('TX', '70', '2586', '244', '1', Array, false)
#10 {main}]
[Tue, 09 Jun 2015 18:36:04 -0600] OUTPUT PDF FILE DOESN'T EXISTS. CLAIM ID: 2586

[Tue, 09 Jun 2015 18:36:06 -0600] ERROR IN $fillManager->fillReport(). CLAIM ID: 2586

[Tue, 09 Jun 2015 18:36:06 -0600] [[o:JRException]:"net.sf.jasperreports.engine.JRException: Unable to get value for field 'claimant_birth_date' of class 'java.sql.Date'" at:
#-19 net.sf.jasperreports.engine.JRResultSetDataSource.getFieldValue(JRResultSetDataSource.java:319)
#-18 net.sf.jasperreports.engine.fill.JRFillDataset.setOldValues(JRFillDataset.java:1330)
#-17 net.sf.jasperreports.engine.fill.JRFillDataset.next(JRFillDataset.java:1231)
#-16 net.sf.jasperreports.engine.fill.JRFillDataset.next(JRFillDataset.java:1207)
#-15 net.sf.jasperreports.engine.fill.JRBaseFiller.next(JRBaseFiller.java:1549)
#-14 net.sf.jasperreports.engine.fill.JRVerticalFiller.fillReport(JRVerticalFiller.java:149)
#-13 net.sf.jasperreports.engine.fill.JRBaseFiller.fill(JRBaseFiller.java:909)
#-12 net.sf.jasperreports.engine.fill.JRBaseFiller.fill(JRBaseFiller.java:822)
#-11 net.sf.jasperreports.engine.fill.JRFiller.fill(JRFiller.java:61)
#-10 net.sf.jasperreports.engine.JasperFillManager.fill(JasperFillManager.java:446)
#-9 net.sf.jasperreports.engine.JasperFillManager.fillReport(JasperFillManager.java:849)
#-8 sun.reflect.GeneratedMethodAccessor5246.invoke(Unknown Source)
#-7 sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
#-6 java.lang.reflect.Method.invoke(Method.java:597)
#-5 php.java.bridge.JavaBridge.Invoke(JavaBridge.java:1044)
#-4 php.java.bridge.Request.handleRequest(Request.java:417)
#-3 php.java.bridge.Request.handleRequests(Request.java:500)
#-2 php.java.bridge.http.ContextRunner.run(ContextRunner.java:145)
#-1 php.java.bridge.ThreadPool$Delegate.run(ThreadPool.java:60)
#0 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(232): java_ExceptionProxyFactory->getProxy(34, 'net.sf.jasperre...', 'T', true)
#1 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(360): java_Arg->getResult(true)
#2 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(366): java_Client->getWrappedResult(true)
#3 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(560): java_Client->getResult()
#4 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(1752): java_Client->invokeMethod(32, 'getCause', Array)
#5 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(2062): java_JavaProxy->__call('getCause', Array)
#6 /home/claim-sftp/webroot/lib/claim_distribution.php(1980): java_exception->__call('getCause', Array)
#7 /home/claim-sftp/webroot/lib/claim_distribution.php(1980): java_InternalException->getCause()
#8 /home/claim-sftp/webroot/lib/claim_distribution.php(1577): ClaimDistribution->claim2pdf('TX', '2586', '244')
#9 /home/claim-sftp/webroot/cron/distribution_queue.php(166): ClaimDistribution->makeDistribution('TX', '70', '2586', '244', '1', Array, false)
#10 {main}]
[Tue, 09 Jun 2015 18:36:06 -0600] ERROR IN $exportManager->exportReportToPdfFile(). CLAIM ID: 2586
[Tue, 09 Jun 2015 18:36:06 -0600] [[o:NullPointerException]:"java.lang.NullPointerException" at:
#-13 java.io.File.<init>(File.java:222)
#-12 net.sf.jasperreports.engine.util.JRLoader.loadObjectFromFile(JRLoader.java:89)
#-11 net.sf.jasperreports.engine.JasperExportManager.exportToPdfFile(JasperExportManager.java:125)
#-10 net.sf.jasperreports.engine.JasperExportManager.exportReportToPdfFile(JasperExportManager.java:485)
#-9 sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
#-8 sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
#-7 sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
#-6 java.lang.reflect.Method.invoke(Method.java:597)
#-5 php.java.bridge.JavaBridge.Invoke(JavaBridge.java:1044)
#-4 php.java.bridge.Request.handleRequest(Request.java:417)
#-3 php.java.bridge.Request.handleRequests(Request.java:500)
#-2 php.java.bridge.http.ContextRunner.run(ContextRunner.java:145)
#-1 php.java.bridge.ThreadPool$Delegate.run(ThreadPool.java:60)
#0 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(232): java_ExceptionProxyFactory->getProxy(37, 'java.lang.NullP...', 'T', true)
#1 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(360): java_Arg->getResult(true)
#2 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(366): java_Client->getWrappedResult(true)
#3 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(560): java_Client->getResult()
#4 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(1752): java_Client->invokeMethod(36, 'getCause', Array)
#5 /opt/jasper/jasperreports-server-cp-5.0.0/apache-tomcat/webapps/JavaBridge/java/Java.inc(2062): java_JavaProxy->__call('getCause', Array)
#6 /home/claim-sftp/webroot/lib/claim_distribution.php(2002): java_exception->__call('getCause', Array)
#7 /home/claim-sftp/webroot/lib/claim_distribution.php(2002): java_InternalException->getCause()
#8 /home/claim-sftp/webroot/lib/claim_distribution.php(1577): ClaimDistribution->claim2pdf('TX', '2586', '244')
#9 /home/claim-sftp/webroot/cron/distribution_queue.php(166): ClaimDistribution->makeDistribution('TX', '70', '2586', '244', '1', Array, false)
#10 {main}]
[Tue, 09 Jun 2015 18:36:06 -0600] OUTPUT PDF FILE DOESN'T EXISTS. CLAIM ID: 2586
